services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -829,6 +813,22 @@ export class NoticeOfWorkApplication extends Component {
               </>
             </Tabs.TabPane>
 
+            <Tabs.TabPane tab="Draft Permit" key="draft-permit" disabled={!verificationComplete}>
+              <>
+                <div className="tab-disclaimer">
+                  <p className="center">
+                    This page contains all the information that will appear in the permit when it is
+                    issued. The Conditions sections is pre-populated with conditions based on the
```

Comment:
"is pre-populated" --> "are pre-populated" ?

Reply:
will update!